vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -466,19 +466,20 @@ - (void)listRecurringTips:(void (NS_NOESCAPE ^)(NSArray<BATPublisherInfo *> *))c
 
 - (void)addRecurringTipToPublisherWithId:(NSString *)publisherId amount:(double)amount
```

Comment:
We should add a completion block here that is called in the `SaveRecurringTip`.  suggested declaration:

`- (void)addRecurringTipToPublisherWithId:(NSString *)publisherId amount:(double)amount completion:(void (^)(BOOL success))completion`

Needs to updated in header as well